pkg/apis/kibana/v1alpha1/kibana_types.go

```diff
@@ -117,9 +116,9 @@ type Kibana struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec      KibanaSpec   `json:"spec,omitempty"`
-	Status    KibanaStatus `json:"status,omitempty"`
-	assocConf *commonv1alpha1.AssociationConf
+	Spec      KibanaSpec                      `json:"spec,omitempty"`
+	Status    KibanaStatus                    `json:"status,omitempty"`
+	assocConf *commonv1alpha1.AssociationConf `json:"-"` //nolint:govet
```

Comment:
Ditto here

Reply:
I think we prefer to keep this hidden like in 0.9.